### ISSUE TITLE
swagger-plugin breaks build

### ISSUE THREAD

**Issue by alefr** [nestjs/nest#4885]
## Current behavior
Tried adding swagger(open api) including the plugin in accordance with the documentation:
https://docs.nestjs.com/recipes/swagger
https://docs.nestjs.com/recipes/swagger#plugin

Building and starting the project with:
`npm run build`
`node dist/main`

Will crash with error:
```
/Users/alefr/Dev/tests/nestjs-demo/dist/movies/movies.controller.js:67
    __metadata("design:paramtypes", [movie_entity_1.Movie]),
                                     ^

ReferenceError: movie_entity_1 is not defined
    at Object.<anonymous> (/Users/alefr/Dev/tests/nestjs-demo/dist/movies/movies.controller.js:67:38)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/Users/alefr/Dev/tests/nestjs-demo/dist/movies/moviesModule.js:13:29)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
```

## Input Code

Made a clone of the project available at:
https://github.com/alefr/nestjs-demo

nest-cli.json
```ts
{
  "collection": "@nestjs/schematics",
  "sourceRoot": "src",
  "compilerOptions": {
    "plugins": ["@nestjs/swagger/plugin"]
  }
}
```

## Expected behavior
Should start the application with swagger enabled.

## Different behavior
If I remove the swagger-plugin:
```ts
{
  "collection": "@nestjs/schematics",
  "sourceRoot": "src"
}
```

and run the same commands:
`npm run build`
`node dist/main`

The application starts with swagger enabled but of course without dto descriptions:
```
[Nest] 71219   - 2020-06-06 18:27:49   [NestFactory] Starting Nest application...
[Nest] 71219   - 2020-06-06 18:27:49   [InstanceLoader] AppModule dependencies initialized +561ms
[Nest] 71219   - 2020-06-06 18:27:49   [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 71219   - 2020-06-06 18:27:49   [InstanceLoader] TypeOrmCoreModule dependencies initialized +16ms
[Nest] 71219   - 2020-06-06 18:27:49   [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 71219   - 2020-06-06 18:27:49   [InstanceLoader] MoviesModule dependencies initialized +0ms
[Nest] 71219   - 2020-06-06 18:27:50   [RoutesResolver] MoviesController {/movies}: +138ms
[Nest] 71219   - 2020-06-06 18:27:50   [RouterExplorer] Mapped {/movies, GET} route +2ms
[Nest] 71219   - 2020-06-06 18:27:50   [RouterExplorer] Mapped {/movies/:id, GET} route +1ms
[Nest] 71219   - 2020-06-06 18:27:50   [RouterExplorer] Mapped {/movies, POST} route +0ms
[Nest] 71219   - 2020-06-06 18:27:50   [NestApplication] Nest application successfully started +2ms
```

## Environment

<pre><code>
Nest version: 7.0.0

 
For Tooling issues:
- Node version: 12.16.1  
- Platform:  Mac 

Others:
OS Version: macOS Catalina 10.15.5

</code></pre>

**Comment by alefr** [comment]
Verified same behavior on:

```
Windows 10.
Nest version: 7.1.3
Node: v12.18.0
```

Not sure if there's something I'm doing wrong or if there's actually a bug.
I'm new to Nest but have tried following the documentation as much as I can.

Looks like something goes wrong with the tsc compilation step when adding the plugin.
I am able to get a build working after manually adding an entity file that goes missing in the transpiled js file with the swagger plugin active: 

`dist/movies/movies.controller.js`:
```
const movie_entity_1 = require("./movie.entity");
```

**Comment by kamilmysliwiec** [comment]
Fixed in `@nestjs/swagger@4.5.10`
